ee/maintained-apps/outputs/santa/darwin.json

```diff
@@ -1,21 +1,21 @@
 {
```

Comment:
TODO diff the scripts

Reply:
Did I forget to add santa as an app when I did https://github.com/fleetdm/fleet/pull/29977?

Reply:
hmm, santa is a dmg.